src/main/resources/raster-catalog-pro.json

```diff
@@ -10,7 +10,7 @@
     },
     {
       "name":"wdpa_protected_areas",
-      "source_uri":"s3://gfw-data-lake/wdpa_protected_areas/v202208/raster/epsg-4326/{grid_size}/{row_count}/iucn_cat/gdal-geotiff/{tile_id}.tif"
+      "source_uri":"s3://gfw-data-lake/wdpa_protected_areas/v202210/raster/epsg-4326/{grid_size}/{row_count}/iucn_cat/gdal-geotiff/{tile_id}.tif"
```

Comment:
Did you mean to make this change?  s3://gfw-data-lake/wdpa_protected_areas/v202210 doesn't seem to exist.  Also, FCD doesn't use this dataset anymore (it uses detailed_wda_protected_areas, which uses the licensed WDPA dataset).